src/schema/generated-schema-types.ts

```diff
@@ -18,6 +18,7 @@ export interface MailItem {
 	invitations?: InviteInfo[] | null;
 	sortField?: string | null;
 	share?: ShareNotification[] | null;
+	rt?: string | null;
```

Comment:
The key `rt` is obscure enough that we should probably normalize it in `src/normalize/entities.ts`, it is called `replyType` in the documentation so could normalize it to that name?